photon-client/src/main/scala/com/linkedin/photon/ml/hyperparameter/HyperparameterDefaults.scala

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright 2018 LinkedIn Corp. All rights reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License. You may obtain a
+ * copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.linkedin.photon.ml.hyperparameter
+
+/**
+ * Hyper-parameter default values
+ */
+object HyperparameterDefaults {
```

Comment:
This whole class seems very problem specific.

Reply:
Yes, this default is for GAME. We are trying to figure out a general config for GAME. If the problem has different numbers of regularizer, we can set others to be zero.

Reply:
Lets call this object GameHyperparameterDefaults to be more explicit.